pkg/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.builds

```diff
@@ -1,12 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <TargetOS Condition="'$(TargetOS)' == '' and '$(OS)' == 'Unix' and Exists('/Applications')">OSX</TargetOS>
```

Comment:
This should rather use `OSGroup` property. We want the build target the OS that it was told on the command, not the OS that it is running on.